src/coreclr/jit/lsra.cpp

```diff
@@ -1641,13 +1650,25 @@ bool LinearScan::isRegCandidate(LclVarDsc* varDsc)
     return true;
 }
 
+void LinearScan::identifyCandidates()
+{
+    if (enregisterLocalVars)
+    {
+        identifyCandidates<true>();
+    }
+    else
+    {
+        identifyCandidates<false>();
+    }
+}
+
```

Comment:
The only caller of `identifyCandidates` is within `buildIntervals` so you could have that callsite use `identifyCandidates<localVarsEnregistered>()` directly.

Reply:
Unfortunately, I had to do go this route for `identifyCandidates()` and `resolveRegisters()` because i was getting linker error. Perhaps I was doing something wrong, and we can chat about it offline.

Reply:
Makes sense. You'd need to move the template definition into a header file. But maybe you can make the outer call FORCEINLINE.